index.html

```diff
@@ -0,0 +1,198 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Construction Engineering global summit</title>
+    <link rel="stylesheet" href="style.css" />
+    <link
+      href="https://cdn.jsdelivr.net/npm/bootstrap@5.3.0-alpha1/dist/css/bootstrap.min.css"
+      rel="stylesheet"
+      integrity="sha384-GLhlTQ8iRABdZLl6O3oVMWSktQOp6b7In1Zl3/Jr59b6EGGoI1aFkw7cmDA6j6gD"
+      crossorigin="anonymous"
+    />
+  </head>
+  <body>
+    <div class="all-container">
+      <section class="background">
+        <header>
+          <nav class="navbar">
+            <a href="index.html" class="nav-branding">CESummit.</a>
+            <ul class="nav-menu">
+              <li class="nav-item">
+                <a href="about.html" class="nav-link">About</a>
+              </li>
+              <li class="nav-item">
+                <a href="#" class="nav-link">Join</a>
+              </li>
+              <li class="nav-item">
+                <a href="#" class="nav-link">Contact</a>
+              </li>
+              <li class="nav-item">
+                <a href="#" class="nav-link">News</a>
+              </li>
+            </ul>
+            <div class="hamburger">
+              <span class="bar"></span>
+              <span class="bar"></span>
+              <span class="bar"></span>
+            </div>
+          </nav>
+        </header>
```

Comment:
- [ ] Please add the dark navbar that exists in the given design in the Home and About pages on the desktop view.

| Your design | Original design |
| --- | --- |
| ![image](https://user-images.githubusercontent.com/62290545/227246197-477817d3-f9d4-4c92-a7e8-831ec6f3613f.png) | ![image](https://user-images.githubusercontent.com/62290545/227246295-d4e6ef31-79e5-488f-920b-c9965d3b408b.png) |